src/chess/position.cc

```diff
@@ -137,4 +142,87 @@ uint64_t PositionHistory::HashLast(int positions) const {
   return HashCat(hash, Last().GetNoCaptureNoPawnPly());
 }
 
+// PrintFen outputs a FEN notation of the board. 
+// based on https://en.wikipedia.org/wiki/Forsyth%E2%80%93Edwards_Notation#Examples
+string Position::GetFen() const  {
+	string result;
+	string enpassant;
+	ChessBoard board = GetBoard();
+	bool fenflipped = false;
```

Comment:
With `GetWhiteBoard()`, no need to flip it and have this variable.

Reply:
Removed fenflipped